.github/workflows/publish-release-artifact.yml

```diff
@@ -0,0 +1,205 @@
+name: Publish artifact
+
+on:
```

Comment:
```suggestion
on:
  workflow_dispatch:
```

Reply:
If assets are not available from get.jenkins.io when the github action is trigger, this would allow us to force build

Reply:
IMHO the most important thing is to be able to force a job